packages/dds/merge-tree/src/partialLengths.ts

```diff
@@ -366,6 +366,8 @@ export class PartialSequenceLengths {
             }
         }
         if (seqPartialLen === undefined) {
+            // len will be assigned below, making this assertion true.
```

Comment:
do we care to rewrite some of these to avoid the error?  e.g. this one could set `seqPartialLen: seqSeglen` then amend it after in the `penultPartialLen` check

Reply:
that said my eyes started glossing over all the instances of this so idk

Reply:
I rewrote a couple trivial ones, but didn't want to spend too much time on it.